server/app.js

```diff
@@ -9,10 +9,8 @@ db.connect(function(err) {
   if (err) throw err;
   db.query('SELECT * FROM reviewText', function(err, result) {
     if (err) throw err;
-    console.log(result);
+    app.get('/party', (req, res) => res.send(result));
```

Comment:
Party time! Will need to eventually make APIs that will allow you to get data from the database using the unique ID number or restaurant name. Proxy url will most likely have the unique ID so that your service can make proper get request.